app/controllers/certificates_controller.rb

```diff
@@ -14,14 +14,25 @@ def create
     end
   end
 
+  def update
```

Comment:
I think we might want to introduce separate `enable` and `disable` actions here, but I think that can follow when we add the new events/

Reply:
Yes, I was planning to have a think about that in the context of the events and how that is triggered from the front-end.

Reply:
Hi guys

Reply:
I thinking missing the kick-off for this has made me not know how we are implementing this I was thinking having a status field and an event that update this

Reply:
https://github.com/alphagov/verify-self-service/tree/ee2PV5rg-model-signing-cert-rotation-ola

Reply:
Yes, we decided to do a PR at this stage to keep the changes smaller. We need to move onto the events. If you can review this PR then I can merge it and move on from there. It's going to get confusing if you are working on another branch on for the same ticket.